compiler/codegen/Relocation.cpp

```diff
@@ -481,6 +481,10 @@ const char *TR::ExternalRelocation::_externalRelocationTargetKindNames[TR_NumExt
    "TR_MethodCallAddress (99)",
    "TR_DiscontiguousSymbolFromManager (100)",
    "TR_ResolvedTrampolines (101)",
+   "TR_InlinedStaticMethod (102",
+   "TR_InlinedSpecialMethod (103)",
+   "TR_InlinedAbstractMethod (104)",
+   "TR_Breakpoint (105)",
```

Comment:
No need to change anything, just noting that putting the enum value inside the hardcoded string is quite fragile and error prone. We should think about removing this and printing it instead (assuming these names are indexed by the enum values).